README.md

```diff
@@ -0,0 +1,70 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+To Do List
+
+Use webpack to bundle JavaScript.
+Learn how to use proper ES6 syntax.
+Use ES6 modules to write modular JavaScript.
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Live Demo (if available)
+
+[Live Demo Link]()
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
```

Comment:
- [ ] Kindly, it would be nice to add explicit steps for cloning your app, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd..`, `npm install` then `npm start`  etc.
- [ ] Make sure to remove all unnecessary titles since they are empty (Prerequisites, run tests Deployment ....) 

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.